cmd/mt-store-cat/chunk.go

```diff
@@ -12,22 +12,22 @@ import (
 // printChunkSummary prints a summary of chunks in the store matching the given conditions, grouped in buckets of groupTTL size by their TTL
 func printChunkSummary(ctx context.Context, store *cassandra.CassandraStore, tables []cassandra.Table, metrics []Metric, groupTTL string) error {
 	now := uint32(time.Now().Unix())
-	end_month := now - (now % cassandra.Month_sec)
+	endMonth := now / cassandra.Month_sec
 
 	for _, tbl := range tables {
 		// per store.FindExistingTables(), actual TTL may be up to 2x what's in tablename.
 		// we query up to 4x so that we also include data that should have been dropped already but still sticks around for whatever reason.
 		start := now - uint32(4*tbl.TTL*3600)
-		start_month := start - (start % cassandra.Month_sec)
+		startMonth := start / cassandra.Month_sec
 		fmt.Println("## Table", tbl.Name)
 		if len(metrics) == 0 {
 			query := fmt.Sprintf("select key, ttl(data) from %s", tbl.Name)
 			iter := store.Session.Query(query).Iter()
 			showKeyTTL(iter, groupTTL)
 		} else {
 			for _, metric := range metrics {
-				for month := start_month; month <= end_month; month += cassandra.Month_sec {
-					row_key := fmt.Sprintf("%s_%d", metric.AMKey.String(), month/cassandra.Month_sec)
+				for num := startMonth; num <= endMonth; num += 1 {
+					row_key := fmt.Sprintf("%s_%d", metric.AMKey.String(), num)
```

Comment:
is `.String()` necessary? I thought it calls that automatically

Reply:
https://golang.org/pkg/fmt/#hdr-Printing basically says that if it implements Error and String that Error will be called first, so we should probably make it a habit to explicitly call String(). I haven't read the entire thing yet so if I find any other information I'll re-post it here.

Reply:
we voted on slack it's better to always be explicit for every type.
see https://github.com/grafana/metrictank/pull/1188